### PR TITLE
Update Outerloop test instructions

### PR DESCRIPTION
**Pull request by ianhays**
Updating instructions with the new way to run outerloop&innerloop  since the merging of https://github.com/dotnet/buildtools/pull/546. Also added a small description of outerloop.

### CODE REVIEW

File: Documentation/building/windows-instructions.md [hunk 1]
```diff
@@ -77,7 +77,13 @@ Tests participate in the incremental build.  This means that if tests have alrea
 The tests can also be filtered based on xunit trait attributes defined in [`xunit.netcore.extensions`](https://github.com/dotnet/buildtools/tree/master/src/xunit.netcore.extensions). These attributes are to be specified over the test method. The available attributes are:
 
 _**`OuterLoop`:**_
-This attribute applies the 'outerloop' category; to run outerloop tests, use the following commandline
+Tests marked as ```Outerloop``` either take longer to execute than normal tests or cover extra cases not in the regular tests. These tests are excluded by default when testing through msbuild but can be enabled manually by adding the  ```/p:Outerloop=true``` property e.g.
```
Comment (stephentoub) [inline Documentation/building/windows-instructions.md]: Sometimes it's that they take longer and cover extra cases.  Other times it's that there's something special about it, e.g. requiring elevated privileges, doing some potentially irreversible mutations to the machine if something goes wrong, requiring a significant amount of resources (e.g. trying to create a 2GB file), etc.

**Comment by stephentoub** [comment]
One nit, otherwise LGTM.

File: Documentation/building/windows-instructions.md (hunk 1 above)
Comment (ianhays) [inline Documentation/building/windows-instructions.md]: I struggled a bit with the best way to phrase it without being too verbose. I figured the "cover extra cases" part would be the umbrella around the scenarios you listed, but maybe not. How about 

```Tests marked as Outerloop are for scenarios that don't need to run every build. They may take longer than normal tests, cover seldom hit code paths, or require special setup or resources to execute. These tests are excluded by default when testing through msbuild but can be enabled manually by adding the...```

Comment (stephentoub) [inline Documentation/building/windows-instructions.md]: Sounds good.

**Pull request merged**